### DISCUSSION TITLE
New release will be worked on!

### DISCUSSION THREAD

**Discussion by metastable-void** [menhera-org/TabArray#417] [Announcements]
It's almost 1 year since our last release.

Now it is a good time to start working on a new release for us, because things to improve are clearer now with many issues submitted, thanks to almost doubled users (thanks a lot)!

I would like to make things arranged for a stable release in August. It'll probably be 12.0.0 because there are many to improve.

(sorry for a little late release schedule!)

**Comment by ejs-tgt** [comment]
Any news?

**Reply by metastable-void** [reply]
Sorry for the delay in development. Due to lack in volunteers, I am solely writing code for CTG, and also maintaining the infrastructure for Menhera.org. I would like to release a new version around on New Year's Day, but code contributions are always welcome.

**Reply by metastable-void** [reply]
Now doing implementation(s) for quickly releasable features!
